Documentation/AEPTarget.md

```diff
@@ -826,4 +841,218 @@ public static func collectLaunchInfo(_ userInfo: [String: Any])
+                        ] 
+                    ], 
+                    "product": [ 
+                        "id": "pId1", 
+                        "categoryId": "cId1" 
+                    ]
+                ], 
+                [
+                    "index": 1, 
+                    "name": "mbox2", 
+                    "parameters": [ 
+                        "mbox_parameter_key2": "mbox_parameter_value2" 
+                    ]
+                ]
+            ]
+        ]
+    ] 
+
+Target.executeRawRequest(request) { responseData, error in 
+    if error != nil { 
+        return 
+    } 
+    guard let responseData = responseData, 
+          !responseData.isEmpty else { 
+        return 
+    } 
+
+    // handle the response   
+
+} 
+```
+
+### Objective C
+
+#### Syntax 
+
+```objectivec
++ (void)executeRawRequest:(nonnull NSDictionary<NSString *, id> *)request completion:(void (nonnull ^)(nullable NSDictionary<NSString *, id> *, nullable NSError *))completion; 
+```
+
+  - *request* : is a dictionary containing prefetch or execute request data in the Target v1 delivery API request format. 
+  - *completion* : is the  the callback which will be invoked with the Target response data or error message after the request is completed.
+
+#### Example
+
+```objectivec
+    NSDictionary *request = @{ 
+        @"execute": @{ 
+            @"mboxes": @[ 
+                @{ 
+                    @"index": @(0), 
+                    @"name": @"mbox1", 
+                    @"parameters": @{ 
+                        @"mbox_parameter_key1": @"mbox_parameter_value1" 
+                    }, 
+                    @"profileParameters": @{ 
+                        @"subscription": @"premium" 
+                    }, 
+                    @"order": @{ 
+                        @"id": @"id1", 
+                        @"total": @(100.34), 
+                        @"purchasedProductIds": @[ 
+                            @"pId1" 
+                        ]
+                    }, 
+                    @"product": @{ 
+                        @"id": @"pId1", 
+                        @"categoryId": @"cId1" 
+                    } 
+                }, 
+                @{ 
+                    @"index": @(1), 
+                    @"name": @"mbox2", 
+                    @"parameters": @{ 
+                        @"mbox_parameter_key2": @"mbox_parameter_value2" 
+                    } 
+
+                } 
+
+            ] 
+
+        } 
+
+    }; 
+
+    [AEPMobileTarget executeRawRequest:request completion:^(NSDictionary<NSString *,id> * _Nullable data, NSError * _Nullable err) { 
+        if (err != nil) { 
+            NSLog(@"Error: %@", err); 
+            return; 
+        } 
+
+        NSLog(@"Target raw response >> %@", data); 
+
+        // handle response 
+    }]; 
+```
+
+
+## sendrawnotification
+This API sends notification request(s) to the configured Target server for display or click notifications. 
```

Comment:
Do we wanna add a sentence here that we set type to either  "click" or  "display" to choose the type of notification

Reply:
Just stating type wouldn't clarify where and how to set the type, also the correct token from a prior prefetch or execute response needed to be supplied as well (options or metric eventToken). The API parameter (request) description states that the payload should conform to the Target v1 delivery request format. This should suffice I believe.